packages/a11y/package.json

```diff
@@ -21,6 +21,7 @@
 	"main": "build/index.js",
 	"module": "build-module/index.js",
 	"react-native": "src/index",
+	"types": "build-types",
```

Comment:
Shouldn’t it be listed in the linter for package.json files in the rule that validates the order of fileds?

Reply:
👍 Added in adb7a78c808f29154e5e356523b8bd0dc9a9d775